source/extensions/filters/http/transformation/transformation_filter.cc

```diff
@@ -65,6 +66,7 @@ Http::FilterDataStatus TransformationFilter::decodeData(Buffer::Instance &data,
   }
 
   if (end_stream) {
+    filter_config_->stats().request_body_transformations_.inc();
```

Comment:
probably want this line in decodeTrailers as well

Reply:
Good call, will add